terraform/environments/nomis/ssm-documents/install-and-manage-script-exporter.yaml

```diff
@@ -0,0 +1,29 @@
+---
+schemaVersion: "2.2"
+description: "Installs script exporter on Linux instances"
+mainSteps:
+- action: aws:downloadContent
+  name: DownloadScriptExporterServiceFile
+  inputs:
+    sourceType: GitHub
+    destinationPath: "/etc/systemd/system/"
+    sourceInfo: "{\"owner\":\"ministryofjustice\", \"repository\":\"modernisation-platform-environments\",\"getOptions\" : \"branch:main\",\"path\" :\"terraform/environments/nomis/ssm-documents/files/script-exporter-service\"}"
+- action: "aws:runShellScript"
+  name: "InstallScriptExporter"
+  inputs:
+    runCommand:
+    - "wget https://github.com/adhocteam/script_exporter/releases/download/v1.2.0/script_exporter-1.2.0.linux-amd64.tar.gz -P /tmp"
+    - "tar -xf /tmp/script_exporter-1.2.0.linux-amd64.tar.gz -C /tmp"
+    - "rm /tmp/script_exporter-1.2.0.linux-amd64.tar.gz"
+    - "cd /tmp/script_exporter-1.2.0.linux-amd64/"
```

Comment:
i'd not cd here, just do mv /tmp/script_exporter-1.2.0.linux-amd64/script_exporter /usr/bin/script-exporter for the next line

Reply:
Removed the change directory statement.